pykeops/tutorials/knn/plot_ivf_numpy.ipynb

```diff
@@ -0,0 +1,467 @@
+{
```

Comment:
I assume that you will eventually replace the .ipynb files with simple Python files?

Reply:
done

format is as discussed